### ISSUE TITLE
bias option of convolution without batch normalization

### ISSUE THREAD

**Issue by MakeDirtyCode** [pytorch/examples#521]
https://github.com/pytorch/examples/blob/87d9a1e930b5b813ba4a7ad49089566a6278eb55/dcgan/main.py#L139

https://github.com/pytorch/examples/blob/87d9a1e930b5b813ba4a7ad49089566a6278eb55/dcgan/main.py#L165

https://github.com/pytorch/examples/blob/87d9a1e930b5b813ba4a7ad49089566a6278eb55/dcgan/main.py#L180

As I know, convolution layer's bias option equal False because it can be adjusted in batch normalization.
So if network do not have a batch normalization after the convolution, then I think it is correct to set bias equal True.
There is definitely a difference in quality when evaluation it.

**Comment by surgan12** [comment]
I think this point makes sense.
In the absence of BN, bias would be good.
